met-web/src/hooks.ts

```diff
@@ -8,15 +8,26 @@ export const useAppDispatch = () => useDispatch<AppDispatch>();
 export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
 
 export const useAppTranslation = () => {
-    const translate = useTranslation();
-    const tenantId = sessionStorage.getItem('tenantId');
+    // Every language has its own default and common namespaces
+    const translate = useTranslation(['default', 'common']);
 
     const { t } = translate;
 
```

Comment:
The language system in MET is inherently a bit confusing. The comments in this file are great so far, and maybe we could add a higher-level one above this function. Would this comment describe the intended behaviour?

```suggestion
/**
* Look to see if a string has a tenant-level translation (if it's related to an engagement, survey, widget, static       
* tenant text, etc.). If none is found, look for a common static text translation found in the common locale file.
**/
```

Reply:
Updated the comments